tests/Integration/Infrastructure/Controller/AbstractTestCase.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Integration\Infrastructure\Controller;
+
+use App\Infrastructure\Persistence\Doctrine\Repository\User\UserRepository;
+use App\Infrastructure\Security\SymfonyUser;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+abstract class AbstractTestCase extends WebTestCase
```

Comment:
Reprendre la formulation "web test case" ?

```suggestion
abstract class AbstactWebTestCase extends WebTestCase
```